### DISCUSSION TITLE
error with amazon_fwds

### DISCUSSION THREAD

**Discussion by shedman214** [moralmunky/Home-Assistant-Mail-And-Packages#871] [Q&A]
I can't figure out how to fix this error. I used my email I signed up with Amazon for the fwd Amazon. I still get this error?
![image](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/32603273/5e1622ad-939f-4de5-ba3b-c4a4729de0c2)

**Comment by firstof9** [comment]
remove the ()s around the email, make sure the email isn't your own email address as well.

**Reply by shedman214** [reply]
Do I make up an email then? The email i use with Amazon is my own person Gmail

**Reply by firstof9** [reply]
Don't enter anything if the email you're checking is the email amazon sends to, that's for if you're forwarding the amazon emails to another email address.

**Reply by shedman214** [reply]
Here is the diagnostics  
[config_entry-mail_and_packages-db987e76ab740ede9fc13c4d17c6c094 (2).json](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/files/14280755/config_entry-mail_and_packages-db987e76ab740ede9fc13c4d17c6c094.2.json)

**Reply by firstof9** [reply]
Ok the config looks fine, what's the errors in `home-assistant.log`?

**Reply by jamesarm97** [reply]
I put my email in by mistake but not doing any forwarding. Now when I try and remove it, it says a string is required.

**Reply by firstof9** [reply]
Try `""`
